extensions/default/src/Panels/PanelStudyBrowser.jsx

```diff
@@ -165,11 +165,11 @@ function PanelStudyBrowser({
     );
     const updatedExpandedStudyInstanceUIDs = shouldCollapseStudy
       ? // eslint-disable-next-line prettier/prettier
-        [
```

Comment:
My understanding is that the prettier plugin is supposed to be run - the spacing changes are caused by that.